lib/pbench/cli/agent/commands/tools/base.py

```diff
@@ -0,0 +1,150 @@
+import pathlib
+
+import click
+
+from pbench.agent.base import BaseCommand
+from pbench.agent import fs
+
+
+class ClearMixIn:
```

Comment:
Should these mix-ins be in separate modules?

Reply:
Why not have the mix-in live in the tool command file instead of piling them all into base?  That way, base only has the common stuff.

Reply:
Removed mixins we can revisit it later.